backend/api/activity/index.js

```diff
@@ -43,5 +65,38 @@ activity.route('/create')
       status: statusCode,
     });
   });
+  
+/**
+ * Searches posts by criteria specified in post data
+ * TODO: Add search criteria
+ * @body {Number} numPosts - The number of posts to return
+ */
+activity.route('/search')
+  .get(async(req, res) => {
+	numPosts = 20;
+	if (Boolean(req.body.numPosts)) {
+		numPosts = req.body.numPosts;
+	}
```

Comment:
These lines can be shortened to one line by using [short circuit assignment](https://dev.to/flexdinesh/short-circuit-assignment-in-javascript--4k80).
`numPosts = req.body.numPosts || 20;`
This works because accessing the property `numPosts` of the body object evaluates to `undefined` if it doesn't hold a value. `undefined` is a "falsey" value so instead of assigning it to `numPosts`, it evaluates the other side of the logical OR operation which evaluates to a "truthy" value and is assigned. If `req.body.numPosts` did contain a value, it would be assigned instead of the value 20.

Reply:
Is that a common thing? If it's not, I would rather stick with what's readable since we're not playing code golf.

Reply:
Yes, short circuit evaluation is a very common thing in JavaScript. It should be easier to read as the comparison and assignment happens all at once. You're essentially saying "if `req.body.numPosts` has a value, assign it. If not, assign 20".